rotkehlchen/api/v1/resources.py

```diff
@@ -959,17 +958,30 @@ def get(
 
 class UniswapBalancesResource(BaseResource):
 
-    get_schema = AsyncGraphQuerySchema()
+    get_schema = AsyncQueryArgumentSchema()
```

Comment:
@LefterisJP back to `AsyncQueryArgumentSchema` (and removed the previous one)

Reply:
thanks